ReactAndroid/src/main/java/com/facebook/react/bridge/JavaScriptModuleRegistration.java

```diff
@@ -46,16 +46,22 @@ public JavaScriptModuleRegistration(Class<? extends JavaScriptModule> moduleInte
     return mModuleInterface;
   }
 
+  private String name_cached;
```

Comment:
Nit: Move this to the top of the class before methods and use camelCase + m prefix. Something like `mName` would be fine.